dev-mode/go.mod

```diff
@@ -0,0 +1,12 @@
+module serverless/dev-mode-extension
+
+go 1.18
+
+require (
+	github.com/Workiva/go-datastructures v1.0.53 // indirect
+	github.com/golang-collections/go-datastructures v0.0.0-20150211160725-59788d5eb259 // indirect
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/sirupsen/logrus v1.9.0 // indirect
```

Comment:
would be good to use zap logger as it is the one we use for other golang packages